canonical-web-and-design/maas-ui
# feat(ui): remove pod selected state/actions/reducers

## Done

- Removed pod selected state, and associated selectors/actions/reducers.
- Updated KVM action forms to only care about the active pod.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Check that the KVM list is unchanged
- Check that you can still open the action forms from the KVM details apge

## Fixes

Fixes #2250
